doc/extract_examples.sh

```diff
@@ -77,4 +77,5 @@ do
     fi
 done
 
+echo "return 0;"
```

Comment:
`main`() always returns 0 implicitly; C11 § 5.1.2.2.3(1):

> If the return type of the `main` function is a type compatible with `int`, [...] reaching the `}` that terminates the `main` function returns a value of 0.

Not strictly necessary to add this, though it's probably good style to.